### DISCUSSION TITLE
Welcome to Projekt-Inzynierski Discussions!

### DISCUSSION THREAD

**Discussion by haribo841** [Projekt-Inzynierski-AK-AS-FD/Projekt-Inzynierski#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by astolowska** [comment]
jeżyce razem silne 💪

**Comment by haribo841** [comment]
Da się zobaczyć na podglądzie co mam niżej? Jak mam ok. powyżej 15 linijek to muszę kompilować, żeby zobaczyć czy dobrze wklepałem formułę, a to spowalnia cały proces dodawania treści. Może istnieje jakiś scroll okna?
![image](https://user-images.githubusercontent.com/48011447/144709224-f447a035-031f-4fd7-a5d2-95b5d74ee87c.png)
Pobierzcie sobie: https://www.sonarlint.org/visualstudio
Akcje zadziałały: https://sonarcloud.io/project/overview?id=Projekt-Inzynierski-AK-AS-FD_Projekt-Inzynierski

**Reply by Filippo1998r** [reply]
Jak dobrze rozumiem o co Ci chodzi, to też miałem ten problem z historią, która się nie pokazywała. Chyba nie ma czegoś takiego jak scroll w projektancie, ale właśnie skumałem, że wystarczy na odpalonym programie zmieniać kod 😆. Chyba musisz tylko zapisać i wtedy zachodzą zmiany.

**Reply by astolowska** [reply]
Nie znam żadnego narzędzia, które mogłoby to umożliwić, niestety. Dzięki za linki i statystyki, super sprawa!

**Reply by haribo841** [reply]
Testowanie na produkcji daje większe okno, podglądy kłamio
![image](https://user-images.githubusercontent.com/48011447/147146523-e78ef58b-6ff6-4fa3-a393-798dc3487ed4.png)

**Reply by haribo841** [reply]
Odpowiedź to: `<ScrollViewer>content_ktory_chemy_moc_przewijac</ScrollViewer>`

**Comment by haribo841** [comment]
Muszę pomyśleć jak to ładnie i równo ułożyć.
![image](https://user-images.githubusercontent.com/48011447/147493591-a2d76557-7da0-40c1-a5d2-776f5a5ccc6e.png)
Zamysł jest taki:
![image](https://user-images.githubusercontent.com/48011447/147493641-7f7da4ff-a551-49d6-aa75-cd12f43b01f0.png)
Pomysły?

**Reply by astolowska** [reply]
Wyrównywanie Latexa to męka, nad której próbami poprawy spędziłam dużo czasu, niemniej dalej nie wiem, jak sprawić by to jakoś wyglądało. Z tego względu, rozwiązaniem, którego używam jest przesunięcie wyrażenia Latexowego niżej:
![image](https://user-images.githubusercontent.com/48011791/147506290-317b3a48-5628-4158-aeeb-48897d94b047.png)
Z formatowaniem takiego układu też może być dużo roboty, szczególnie przy zdecydowaniu się na centrowanie, ale działa.

**Reply by haribo841** [reply]
Bardziej chodzi mi o te myślniki

**Reply by astolowska** [reply]
W takim razie rozbicie tekstu na kilka TextBlocków i wrzucenie w jednę grupę/zestackowanie byłoby chyba najbardziej optymalne, wtedy powinny prezentować się normalnie.

**Reply by haribo841** [reply]
Na razie są rozbite. Każdy paragraf od myślnika to osobny TextBox, jeśli chodzi o wysokość myślnika z równania, to wszystko gra, tylko wysokość się nie zgadza i nie wiem jak to ułożyć

**Reply by astolowska** [reply]
Spróbuję zajrzeć i doradzić.

**Reply by haribo841** [reply]
Odpowiedź to: 
```
<InlineUIContainer>
    <cośtam Height="liczba"></cośtam >
</InlineUIContainer>
```
